src/blk/kernel/KernelDevice.cc

```diff
@@ -82,6 +82,21 @@ KernelDevice::KernelDevice(CephContext* cct, aio_callback_t cb, void *cbpriv, ai
   }
 }
 
+int KernelDevice::choose_fd(FAMode buffered, int write_hint) const
```

Comment:
any reason to shuffle this method around?

Reply:
@tchaikov 
I like to have define before use if possible.
And it was now just at an odd place in the current code

Reply:
@wjwithagen i'd suggest keep where it is unless we have good reason to move it to a new place.